docs/source/getting-started.md

```diff
@@ -0,0 +1,110 @@
+
+
+### Hello World
+Let's write a quick Hello World script that has a method that will save `"Hello World"` on the blockchain 
+and another method that will return the string.
+
+Those 2 functions will need to be callable and will also need to change the values inside the storage, 
+so let's import both the `public` decorator and the `storage` package.
+
+```python
+# hello_world.py
+from boa3.builtin.compile_time import public
+from boa3.builtin.interop import storage
+
+
+@public     # the public decorator will make this method callable
+def save_hello_world():             # an empty return type indicates that the return is None
+    storage.put("first script", "Hello World")      # the put method will store the "Hello World" value with the "first script" key
+
+
+@public     # the public decorator will make this method callable too
+def get_hello_world() -> str:       # this method will return a string, so it needs to specify it
+    return storage.get("first script")              # the get method will return the value associated with "first script" key
+```
+
+Alternatively, instead of creating a `save_hello_world` function and having to call it yourself, you could instead create
+a public `_deploy` function and it will automatically be called when you deploy your smart contract into the blockchain.
+
+```python
+# hello_world.py
+from typing import Any
+
+from boa3.builtin.compile_time import public
+from boa3.builtin.interop import storage
+
+
+@public
+def _deploy(data: Any, update: bool):               # the _deploy function needs to have this signature
+    storage.put("first script", "Hello World")      # "Hello World" will be stored when this smart contract is deployed
+
+
+@public
+def get_hello_world() -> str:       
+    return storage.get("first script")              
+```
+
+## Compiling your Smart Contract
+
+### Using CLI
+
+```shell
+$ neo3-boa path/to/your/file.py
```

Comment:
we need to ensure that we change here to the following when we release the new documentation.
```suggestion
$ neo3-boa compile path/to/your/file.py
```
On master branch and release versions we still use `neo3-boa` command, but on development branch `neo3-boa compile` is already implemented